### ISSUE TITLE
currency name display error

### ISSUE THREAD

**Issue by YuanYuanOwO** [divios/DailyShop#133]
Hi,i am using gemseconomy,and i got a issue
````yaml
 GREEN_CONCRETE_POWDER:
    material: GREEN_CONCRETE_POWDER
    buyPrice:
      fixed: 115.5
    rarity: Common
    econ:
      name: gemsEconomy
      currency: 金币
    confirm_gui: true
````  
it's should be display `金币` but in game like this
![Gemsbug.png](https://tu.tianji.link/images/2022/04/03/1AZCKR_ELI5G7NO4A.png)
Trade is normal,just a display error

**Comment by divios** [comment]
Hi! This must be an issue with UTF-8 encoding. I will fix it in the next update

**Comment by divios** [comment]
Hi! Sorry for the late reply. [This build](https://we.tl/t-M5xGDIr3zj) fixes some issues with how .yml files were handled. Let me know if it fixes this, but it should.

**Comment by YuanYuanOwO** [comment]
> Hi! Sorry for the late reply. [This build](https://we.tl/t-M5xGDIr3zj) fixes some issues with how .yml files were handled. Let me know if it fixes this, but it should.

it's fixed!
thank you
